package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "asam-osi-types",
-  "version": "3.7.0",
+  "name": "@lichtblick/asam-osi-types",
+  "version": "0.1.0",
```

Comment:
How would you suggest we treat this version in here? Basically the repository version and we use this version for the release tag, right? The package version is temporarly changed on the deploy to npm then

Reply:
Yes, I suggest we treat this version locally in the project, this will keep track of the internal changes of the project, but the final version reflected in the package will always reflect the OSI submodule.